activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/java/org/activiti/cloud/services/modeling/validation/extensions/ProcessExtensionsModelValidator.java

```diff
@@ -93,14 +90,26 @@ public ProcessExtensionsModelValidator(SchemaLoader processExtensionsSchemaLoade
     protected Stream<ModelValidationError> validateBpmnModel(Model model,
                                                                    ValidationContext validationContext,
                                                                    BpmnProcessModelContent bpmnModel) {
+        return bpmnModel.getBpmnModel().getProcesses().stream()
```

Comment:
can it be changed to a more readable fashion? Too many map() and nested lambdas makes difficult to read

Maybe wrap part of the code to a private method that helps to indicate what that part of code does